### PR TITLE
release/v0.16.0

### PR DESCRIPTION
**Pull request by bdougie**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [ ] ✨ Feature
- [ ] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 📝 Documentation Update
- [x] 🔖 Release
- [ ] 🚩 Other

## Description

Work no included in this list should not PR'd here. 
https://github.com/open-sauced/open-sauced/issues?q=is%3Aopen+is%3Aissue+milestone%3Av0.16.0

If there is a feature you would like to see shipped, please open an issue and have me or the @open-sauced/maintainers team mark it for the next release. 

## QA
This is a release branch. Though code review is valuable, actually QA testing is preferred. All commits are from review PRs, it just important to test the combination of new features have not broken.

[Stagin version of release/v0.16.0](https://deploy-preview-719--open-sauced.netlify.app/)

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [x] 🙋 no, this is a release and already has tested and approved code.

## Added to documentation?

- [ ] 📜 readme
- [x] 🙅 this is a release and already has features that should have been documented prior.

### CODE REVIEW

File: src/styles/Card.js [hunk 1]
```diff
@@ -71,6 +71,39 @@ const FittedCard = styled(Card)`
   padding: 0;
 `;
 
+const ButtonBoard = styled.div`
+  flex: 1;
+<<<<<<< HEAD
```
Comment (yg) [inline src/styles/Card.js:76]: @bdougie Merge conflict :thinking:

**Comment by bdougie** [comment]
@open-sauced/triage if you have time can you check out the deploy preview to confirm the features merged all work properly

**Review by filiptronicek — COMMENTED** [review]
**Edit: this has since been resolved. My goals repo was outdated and private**

Is this supposed to change automatically or is it static? Because if it is, I am not much of a fan of this, due to it kinda ruining the cleanness of the dashboard.

this
image
 looks much better than
image
even if it is just a placeholder, I don't think it should be included in the release. Maybe something I am not getting here?

**Comment by yg** [comment]
> Is this supposed to change automatically or is it static? Because if it is, I am not much of a fan of this, due to it kinda ruining the cleanness of the dashboard.

@filiptronicek This is dynamic when CI finish adding the data to the cache repo, the sync in progress will be hidden and star count will get displayed.

From the design part, this is just an MVF (Minimum viable feature), we can update in upcoming PRs!

WDYT?

**Comment by filiptronicek** [comment]
Sure, makes perfect sense, thanks @yg!

**Review by filiptronicek — APPROVED** [review]
LGTM, less ship it!

**Pull request merged**